test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -322,7 +371,7 @@ protected void destroyState(State<S> state, @Nullable Throwable cause) throws Th
     @Test
     public void test()
     {
-        Property.StatefulBuilder statefulBuilder = stateful().withSteps(20).withStepTimeout(Duration.ofMinutes(2)).withExamples(1);
+        Property.StatefulBuilder statefulBuilder = stateful().withSteps(20).withStepTimeout(Duration.ofMinutes(3)).withExamples(1);
```

Comment:
some startup tasks were hitting this often... moving to 3m made the test more stable... why a cluster with 10 writes needs 2 minutes to startup was not solved... but there are def existing issues making it slow (such as nodes retry spamming loading range transactions without making progress...)